src/picasso-definition/components/out-of-bounds/index.js

```diff
@@ -0,0 +1,64 @@
+import KEYS from '../../../constants/keys';
+
+const OOB_SPACE = 12;
```

Comment:
OOB_SPACE should be 10 to make it the same as old scatter plot in sense client

Reply:
Yes, I have increased OOB_SPACE because when it looked too close to the axis. See pictures below, comparing old and new scatterplot, both for oob_space 10 and 12.
OOB_SPACE = 10
<img width="1769" alt="oobspace_10" src="https://user-images.githubusercontent.com/13185716/133757274-f68a2229-038e-47ff-b2b5-4f7765d2fad6.png">

OOB_SPACE = 12
<img width="1769" alt="oobspace_12" src="https://user-images.githubusercontent.com/13185716/133757488-99cb2b9b-d655-4a8f-8674-ee9bb1647a03.png">